tpm/tpm.go

```diff
@@ -61,6 +61,21 @@ func OpenTPM(path string) (io.ReadWriteCloser, error) {
 	return rwc, nil
 }
 
+// GetKeys gets the list of handles for currently-loaded TPM keys.
+func GetKeys(f *os.File) ([]Handle, error) {
```

Comment:
In the other functions in this file, I've been using the more generalized io.ReadWriter interface instead of *os.File. This makes it easier to build a fake TPM for testing.